lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
@marcparadise What version of chef (chef-server 12 package) does this endpoint exist? @jkeiser suggestion gets me passed the JSON error but I am hitting a 404 exception...

```
[2015-02-27T06:30:29+00:00] DEBUG: ---- HTTP Status and Header Data: ----
[2015-02-27T06:30:29+00:00] DEBUG: HTTP 1.1 200 OK
[2015-02-27T06:30:29+00:00] DEBUG: server: ngx_openresty/1.4.3.6
[2015-02-27T06:30:29+00:00] DEBUG: date: Fri, 27 Feb 2015 06:30:29 GMT
[2015-02-27T06:30:29+00:00] DEBUG: content-type: application/json
[2015-02-27T06:30:29+00:00] DEBUG: transfer-encoding: chunked
[2015-02-27T06:30:29+00:00] DEBUG: connection: close
[2015-02-27T06:30:29+00:00] DEBUG: x-ops-api-info: flavor=cs;version=12.0.0;oc_erchef=1.2.0
[2015-02-27T06:30:29+00:00] DEBUG: content-encoding: gzip
[2015-02-27T06:30:29+00:00] DEBUG: ---- End HTTP Status/Header Data ----
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::RemoteRequestID#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Authenticator#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Decompressor#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: decompressing gzip response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::CookieManager#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::JSONOutput#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::JSONInput#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::JSONInput#handle_request
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::JSONOutput#handle_request
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::CookieManager#handle_request
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Decompressor#handle_request
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Authenticator#handle_request
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::RemoteRequestID#handle_request
[2015-02-27T06:30:29+00:00] DEBUG: Initiating POST to https://poliva-chef12-standalone-thor.poliva.dev.altus.bblabs/organizations/blackberry/users
[2015-02-27T06:30:29+00:00] DEBUG: ---- HTTP Request Header Data: ----
[2015-02-27T06:30:29+00:00] DEBUG: Content-Type: application/json
[2015-02-27T06:30:29+00:00] DEBUG: Accept: application/json
[2015-02-27T06:30:29+00:00] DEBUG: Accept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3
[2015-02-27T06:30:29+00:00] DEBUG: X-REMOTE-REQUEST-ID: fc7ea78f-236d-4d9a-a5ff-ef3bd7ddaafa
[2015-02-27T06:30:29+00:00] DEBUG: Content-Length: 21
[2015-02-27T06:30:29+00:00] DEBUG: ---- End HTTP Request Header Data ----
[2015-02-27T06:30:29+00:00] DEBUG: ---- HTTP Status and Header Data: ----
[2015-02-27T06:30:29+00:00] DEBUG: HTTP 1.1 404 Not Found
[2015-02-27T06:30:29+00:00] DEBUG: server: ngx_openresty/1.4.3.6
[2015-02-27T06:30:29+00:00] DEBUG: date: Fri, 27 Feb 2015 06:30:29 GMT
[2015-02-27T06:30:29+00:00] DEBUG: content-type: text/html; charset=utf-8
[2015-02-27T06:30:29+00:00] DEBUG: transfer-encoding: chunked
[2015-02-27T06:30:29+00:00] DEBUG: connection: close
[2015-02-27T06:30:29+00:00] DEBUG: status: 404 Not Found
[2015-02-27T06:30:29+00:00] DEBUG: strict-transport-security: max-age=31536000
[2015-02-27T06:30:29+00:00] DEBUG: x-request-id: 19a9f557b9f7b991e599a2cf7b6addee
[2015-02-27T06:30:29+00:00] DEBUG: x-runtime: 0.004863
[2015-02-27T06:30:29+00:00] DEBUG: content-encoding: gzip
[2015-02-27T06:30:29+00:00] DEBUG: ---- End HTTP Status/Header Data ----
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::RemoteRequestID#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Authenticator#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Decompressor#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: decompressing gzip response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::CookieManager#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::JSONOutput#handle_response
[2015-02-27T06:30:29+00:00] DEBUG: Expected JSON response, but got content-type 'text/html; charset=utf-8'
[2015-02-27T06:30:29+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::JSONInput#handle_response
[2015-02-27T06:30:29+00:00] INFO: HTTP Request Returned 404 Not Found:


    ================================================================================
    Error executing action `create` on resource 'chef_organization[blackberry]'
    ================================================================================
```